### PR TITLE
fix validate connection failing

### PR DESCRIPTION
**Pull request by ChenyuLInx**
resolves #

### Description
make sure validate connection won't fail due to profile doesn't have comment

### Checklist

- [x] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
- [x] I have run this code in development and it appears to resolve the stated issue
- [x] This PR includes tests, or tests are not required/relevant for this PR
- [ ] I have updated the `CHANGELOG.md` and added information about my change to the "dbt-bigquery next" section.

### CODE REVIEW

**Comment by McKnight-42** [comment]
https://github.com/dbt-labs/dbt-bigquery/runs/6187865236?check_suite_focus=true looks like last night we had a failure in regards to this? do we know why it's happening?

**Comment by gshank** [comment]
Yes, that's related to this. There was a PR in dbt-core which requires this pull request in BigQuery.

File: dbt/adapters/bigquery/connections.py [hunk 1]
```diff
@@ -380,8 +380,11 @@ def raw_execute(self, sql, fetch=False, *, use_legacy_sql=False):
         client = conn.handle
 
         fire_event(SQLQuery(conn_name=conn.name, sql=sql))
-
-        if self.profile.query_comment and self.profile.query_comment.job_label:
+        if (
+            hasattr(self.profile, "query_comment")
```
Comment (ChenyuLInx) [inline dbt/adapters/bigquery/connections.py:384]: This is the only meaningful change, others are just format fixes by black

**Review by gshank — APPROVED** [review]

**Review by McKnight-42 — APPROVED** [review]
LGTM

**Pull request merged**